Src/AutoFixture/Dsl/NodeComposer.cs

```diff
@@ -411,7 +412,7 @@ public IPostprocessComposer<T> WithAutoProperties()
             Expression<Func<T, TProperty>> propertyPicker)
         {
             var m = propertyPicker.GetWritableMember().Member;
-            if (m.ReflectedType != typeof(T))
+            if (m.DeclaringType != typeof(T))

```

Comment:
Thank you; I didn't know that. Apparently, this change isn't an issue, since all tests still pass.